content/help/sso/steamconnect.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Steam Connect
+tags:
+- Features
+- Single Sign-On
+- SteamConnect
+category: help
+menu:
+  help:
+    parent: sso
+    identifier: steamconnect
+    weight: 2
+aliases:
+- /features/sso/steamconnect
+---
+
+## SteamConnect : Overview
```

Comment:
We don't need to SteamConnect in all of the titles here. The title from the metadata at the top of the front-matter will get used as the pages title already. As a personal choice for the titles I don't particularly care, just figured I'd mention it.